screenpy/narration/adapters/stdout_adapter.py

```diff
@@ -7,12 +7,15 @@
 from functools import wraps
 from typing import Any, Callable, Generator, List, Optional
 
-from screenpy import settings
+from screenpy.settings import StdOutAdapterSettings
 
 from ..gravitas import AIRY, EXTREME, HEAVY, LIGHT, NORMAL
 
 # pylint: disable=unused-argument
 
+# initialize adapter settings
+settings = StdOutAdapterSettings()
+
```

Comment:
Doesn't making two different `Settings` mean users wouldn't have the ability to change the values in one place?  e.g. conftest

Reply:
Oh yeah, i meant to put conversation points in here but forgot.

My intention here is that this `StdOutAdapter` should mimic the other adapter's placements, where this one's `config` (also woops i should rename it) exists alongside the Adapter itself. I was wondering if i should put the actual class in here, in its own `settings.py` file, too.

If the user wanted to change the config outside of `pyproject.toml` or the environment variables for both of these, then yeah they'd have to do something like:

```python
from screenpy import config as screenpy_config
from screenpy.narration.adapters import config as stdoutadapter_config

screenpy_config.TIMEOUT = 50
stdoutadapter_config.INDENT_CHAR = ">"
```

Reply:
Just to close the loop in the PR, too—on Discord, we came up with a nicer approach:

```python
import screenpy
from screenpy.narration import stdout_adapter

screenpy.config.TIMEOUT = 50
stdout_adapter.config.INDENT_CHAR = ">"
```